src/machine/board_esp32c3-supermini.go

```diff
@@ -0,0 +1,57 @@
+//go:build esp32c3_supermini
+
+// This file contains the pin mappings for the ESP32 supermini boards.
+//
+// - https://forum.arduino.cc/t/esp32-c3-supermini-pinout/1189850/7
```

Comment:
```suggestion
// - https://www.tindie.com/products/adz1122/esp32-c3-development-board-esp32-supermini/
// - https://forum.arduino.cc/t/esp32-c3-supermini-pinout/1189850/7
```

Reply:
What about using "https://dl.artronshop.co.th/ESP32-C3%20SuperMini%20datasheet.pdf" instead of a webstore? This is also a option they refer to multiple sources: https://www.sudo.is/docs/esphome/boards/esp32c3supermini/

Reply:
Yes, sure, great! I just wanted to find some "origin" not an obscure forum post.
BTW, they seem had an official github repo some time ago (referenced on that shop page), but it is now no more...
Is the board still widely available and produced/supported?

Reply:
As far as I know yes. Especially in China. Saw them on multiple sites, I recently got 6 of them in the mail.